rest-api/dao/biobank_order_dao.py

```diff
@@ -265,8 +276,43 @@ def from_client_json(self, resource_json, participant_id=None, client_id=None):
           % (participant_id, resource.subject, self._participant_id_to_subject(participant_id)))
     self._add_identifiers_and_main_id(order, resource)
     self._add_samples(order, resource)
+    if resource.status and not resource.amendedInfo:
```

Comment:
What if there's amendedInfo and resource.status has changed? If that's not valid, we should probably throw an exception... if it is valid, we should handle it.

Reply:
OK. Doesn't this apply to amendedInfo, too?

I guess I'm confused... you don't actually call this method on PATCH ever, right? So I'm not sure why you need to handle new statuses here, if we only do cancel / restore on PATCH calls?

Reply:
You're correct, at one time this was called here (before using `handling_info`). I don't think this is required at all now. Removed.